src/www/ui/template/browse_file.js.twig

```diff
@@ -45,7 +45,7 @@ function createDirlistTable() {
             }
           }
         },
-        {"sTitle":"Files Cleared","sClass":"center","bSortable":false,"bSearchable":false},
+        {"sTitle":"Files Cleared / Not-Cleared / Total","sClass":"center","bSortable":false,"bSearchable":false},
```

Comment:
It will be better to have the title as **"Cleared / Open / Total"**. It will help reducing the amount of text...

Reply:
Done.